composer.json

```diff
@@ -4,7 +4,10 @@
     "description": "A security checker for your composer.lock",
     "license": "proprietary",
     "require": {
-        "php": ">=7.3"
+        "php": ">=7.3",
+        "symfony/console": "^5.0",
+        "symfony/options-resolver": "^5.0",
+        "symfony/process": "^5.0"
```

Comment:
Only allowing Symf5 and hooking this into our normal process will essentially kill Symf4 support.
I don't think we should do this if it can be avoided.